Makefile

```diff
@@ -0,0 +1,26 @@
+export SHELL:=/bin/bash
+KONG_DIST_KUBERNETES_VERSION?=master
+KONG_BUILD_TOOLS_VERSION?=master
+
+setup_tests:
+	-rm -rf kong-build-tools
```

Comment:
Why delete it just to download it again?

Reply:
setting up the extra repositories needs to handle first run as well as subsequent runs. I think it's probably fine with either resetting to a clean slate (via `rm`) or making sure we refresh from origin (via `fetch`).

Reply:
The fetch and reset will be sufficient. Could even try to reset first and if that fails then fetch just a specific commit.

Reply:
I don't think a fetch / reset will work if there are changes in the dependent project

Reply:
Why wouldn't it?

Reply:
I tested locally and it wasn't reliable